source/client/Controller.py

```diff
@@ -413,6 +415,14 @@ def Network_connectionDenied(self, data):
         connection.Close()
 
     ### Gameplay messages ###
+
+    def Network_defineGame(self, data):
+        #todo: is this if statement necessary?
```

Comment:
You could just call the import from here instead of calling it in the runClient loop as it is now.

Reply:
The if is not REQUIRED, but a check you have the right amount of info is never WRONG, just remember to raise a clear exception in the else case so it is obvious what happened if you ever hit it.

Reply:
Moved import as suggested.   Replaced existing print statement with an exception.